OtripleS.Web.Api/Services/Attchments/IAttachmentService.cs

```diff
@@ -11,6 +11,7 @@ namespace OtripleS.Web.Api.Services.Attachments
 {
     public interface IAttachmentService
     {
+        ValueTask<Attachment> InsertAttachmentAsync(Attachment attachment);
```

Comment:
it should be "add" instead of "insert"

Reply:
@viralpandya I thought that we have to name on the service layer the methods like we do in real life (following @hassanhabib advice on one of his videos) e.g. we have named `RegisterStudentAsync` not `AddStudentAsync` because on service layer we are registering student and on broker level we are adding student. Thats why i named it `InsertAttachmentAsync` and not `AddAttachmentAsync`.

Reply:
@viralpandya & @eriadhami this is an interesting case where the SQL language and the business language are similar.
I have seen some folks use the term insert attachment when the attachment is going to be a part of the body of an e-mail or document.
But people also say add attachment to have a more generic meaning of whether it's going to be included in the body or just be an attachment outside of the body of a message for instance.
The goal here is to distinguish between the service language and the broker language, in this case here I think it's better to go with `AddAttachmentAsync` or `SaveAttachmentAsync` either way is fine.

Reply:
@viralpandya and @hassanhabib name changed to `AddAttachmentAsync`